cli/Makefile

```diff
@@ -11,9 +11,15 @@ GO_FLAGS += -trimpath
 GO_FILES = $(shell find . -name "*.go")
 SRC_FILES = $(shell find . -name "*.go" | grep -v "_test.go")
 
-turbo: $(SRC_FILES) go.mod go.sum
+turbo: compile-protos $(SRC_FILES) go.mod
 	CGO_ENABLED=0 go build $(GO_FLAGS) ./cmd/turbo
 
+compile-protos: internal/turbodprotocol/turbod.proto
+	protoc --go_out=. --go_opt=paths=source_relative \
+    --go-grpc_out=. --go-grpc_opt=paths=source_relative \
+    internal/turbodprotocol/turbod.proto
+# touch compile-protos
```

Comment:
If that's your goal, rather than creating a fake thing I think we should try to leverage `make` a bit differently. I think we can do it if we make it delegate.

```make
compile-protos: internal/turbodprotocol/turbod.pb.go internal/turbodprotocol/turbod_grpc.pb.go

internal/turbodprotocol/turbod.pb.go: internal/turbodprotocol/turbod.proto
  make protoc

internal/turbodprotocol/turbod_grpc.pb.go: internal/turbodprotocol/turbod.proto
  make protoc

protoc: internal/turbodprotocol/turbod.proto
  protoc --go_out=. --go_opt=paths=source_relative \
    --go-grpc_out=. --go-grpc_opt=paths=source_relative \
    internal/turbodprotocol/turbod.proto
```